solidity/package.json

```diff
@@ -56,15 +53,14 @@
     "bn.js": "^4.11.8",
     "chai": "^4.2.0",
     "eslint": "^5.16.0",
-    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git#0.2.0",
-    "eslint-config-prettier": "^6.10.0",
+    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git#0.3.0",
     "eth-gas-reporter": "^0.2.22",
     "ethlint": "^1.2.4",
     "ganache-cli": "^6.4.3",
     "mocha": "^7.0.1",
     "moment": "^2.27.0",
-    "prettier": "^1.19.1",
-    "prettier-plugin-solidity": "^1.0.0-alpha.40",
+    "prettier": "^2.0.2",
+    "prettier-plugin-solidity": "^1.0.0-beta.10",
```

Comment:
As we're not running prettier in `lint:sol` do we still need this dependency?

Reply:
We should still be running `prettier` directly on the Solidity files, as I don't believe `eslint` will pick those up.